src/Entities/CollectionFeaturedArtwork.ts

```diff
@@ -0,0 +1,52 @@
+import { Field, ID, Int, ObjectType } from "type-graphql"
+import { Column, Entity, ObjectID, ObjectIdColumn } from "typeorm"
+
+@ObjectType()
+@Entity()
+export class CollectionFeaturedArtworkImage {
```

Comment:
I think we would just name this model `ArtworkImage` as we'll end having to maintain two models that have essentially the same properties